cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
`INTERPRETED` also hits PHP and the `HALF_COMPILED` languages haven't seen any testing either -- so I'd say we're probably not too much further down the track than we were.